mix.exs

```diff
@@ -33,8 +33,7 @@ defmodule Finch.MixProject do
 
   defp deps do
     [
-      {:mint, "~> 1.6"},
-      {:castore, "~> 0.1 or ~> 1.0"},
+      {:mint, "~> 1.6.2 or ~> 1.7"},
```

Comment:
Mint is using OS trust store since 1.6.1. I thought I'd require the latest available version at the moment, 1.6.2.